Doc/library/functions.rst

```diff
@@ -541,6 +541,11 @@ are always available.  They are listed here in alphabetical order.
    :returns: The result of the evaluated expression.
    :raises: Syntax errors are reported as exceptions.
 
+   .. warning::
+
+      This function executes arbitrary code. Calling it with
+      user-supplied input may lead to security vulnerabilities.
+
```

Comment:
Maybe directly refer to `ast.literal_eval()` here? You need to remove the sentence further down though.

```suggestion
      user-supplied input may lead to security vulnerabilities.
      See :func:`ast.literal_eval` for a function that can safely evaluate strings
      with expressions containing only literals.

```

Reply:
Let's do this in a future PR if desired.